### ISSUE TITLE
Support Swagger 2

### ISSUE THREAD

**Issue by afeld** [solso/source2swagger#20]
Someone had added this task to a list of potential projects for a first-time open source contributor workshop I was running. I'm doing another one [today](http://www.meetup.com/hackerhours/events/226799942/), so just wanted to create a placeholder for this task. Thanks!

https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md

**Comment by solso** [comment]
sure no problem, hope you had a good workshop

**Comment by afeld** [comment]
Did you mean to close the issue? Does it have swagger 2 support yet?

**Comment by solso** [comment]
nope, have not checked what's new with swagger 2 to see how difficult would it be, re-opening
